x-pack/plugins/security_solution/server/lib/siem_migrations/rules/task/agent/graph.ts

```diff
@@ -46,7 +50,7 @@ export function getRuleMigrationAgent({
 
 const matchedPrebuiltRuleConditional = (state: MigrateRuleState) => {
   if (state.elastic_rule?.prebuilt_rule_id) {
-    return END;
+    return 'end';
```

Comment:
Why not use `END`?

Reply:
Its because I updated the valid return keys (expected to be string) when the `addConditionalEdges` is defined above.